File: 2058-concatenation-of-array/concatenation-of-array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;

        int [] ans = new int[n+n];
        int j = 0;
        while(j < n*2){
            if(j < n) ans[j] = nums[j];
            else ans[j] = nums[j-n];
            j++;
        }
        return ans;
    }
}

File: 2058-concatenation-of-array/concatenation-of-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getConcatenation = function(nums) {
        let ans = [];
        let j = 0;
        while(j < nums.length*2){
            if(j < nums.length) ans.push(nums[j]);
            else ans.push(nums[j-nums.length]);
            j++;
        }
        return ans;
};